notebooks/hello_world_thunderfx.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>"gradient" -&gt; "training"</p><p> </p><p>"Training" is the practitioner-facing scenario, and the natural companion to "inference"</p><p> </p><p>Would</p><p> </p>

<pre class="ql-syntax" spellcheck="false">...; r.sum().backward(); ...
</pre><p> </p><p>work here? That might be more familiar to people than the explicit autograd invocation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>